crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -510,6 +558,14 @@ mod test {
                     .enabled(true)
                     .build(),
             )
+            .with_file_output(
+                Settings::builder()
+                    .with_environment_variable("ABC_FILE")
+                    .with_default_level(LevelFilter::INFO)
+                    .enabled(true)
+                    .file_log_settings_builder(String::from("/abc_file_dir"))
```

Comment:
question: While this technically works, can we use `PathBuf` here instead of `String`? This helps to show developers the preferred way of using this API.

Reply:
```suggestion
                    .file_log_settings_builder(PathBuf::from("/abc_file_dir"))
```